lib/src/center.dart

```diff
@@ -1,5 +1,32 @@
+import 'package:turf/bbox.dart';
 import 'package:turf/helpers.dart';
 
-Position center(BBox bBox) {
-  return Position.named(lat: (bBox.lat1 + bBox.lat2) / 2, lng: (bBox.lng1 + bBox.lng2) / 2);
+/// Takes a {@link Feature} or {@link FeatureCollection} and returns the absolute center point of all features.
+///
+/// @name center
+/// @param {GeoJSON} geojson GeoJSON to be centered
+/// @param {Object} [options={}] Optional parameters
+/// @param {Object} [options.properties={}] Translate GeoJSON Properties to Point
+/// @param {Object} [options.bbox={}] Translate GeoJSON BBox to Point
+/// @param {Object} [options.id={}] Translate GeoJSON Id to Point
+/// @returns {Feature<Point>} a Point feature at the absolute center point of all input features
+/// @example
+/// var features = turf.points([
+///   [-97.522259, 35.4691],
+///   [-97.502754, 35.463455],
+///   [-97.508269, 35.463245]
+/// ]);
+///
+/// var center = turf.center(features);
+///
+/// //addToMap
+/// var addToMap = [features, center]
+/// center.properties['marker-size'] = 'large';
+/// center.properties['marker-color'] = '#000';
+Feature<Point> center<P>(GeoJSONObject geoJSON, Map<String, dynamic> options) {
```

Comment:
is it actually needed to use `Map<...> options` here and then call `options['properties']` to access the properties? Could we not simply just put `Map<String, dynamic> properties` in the signature of the function here?